MIGRATION_v4x.md

```diff
@@ -0,0 +1,158 @@
+# Migration Guide
+
+**Table of Contents**
+
+- [MUI 5 Migration](#mui-5-migration)
+- [Packages Changes](#packages-changes)
+  - [Visualizations package](#visualizations-package)
+  - [Compatibility package](#compatibility-package)
+- [Components](#components)
+- [What's next?](#whats-next)
+
+# From v3 to v4
+
+Having reached a point of stability and maturity we have approached v4 mostly as a **technical release**, with a focus on cleanup and organization in order to **unlock the next stage of growth** (streamline the adoption of new versions of the Design System and improve the customization experience).
+
+## MUI 5 Migration
+
+We have been following [MUI](https://mui.com/) for a long time and using many of the solutions provided since day one. Our approach has always been to use what we find value and unlock the remaining features for our users. Following the same spirit we decided that it was important to upgrade to the latest MUI v5.
```

Comment:
Suggestion: We have been following [MUI](https://mui.com/) for a long time and using many of the solutions provided since day one. Our approach has always been to use what we find _**valuable and enable access to the remaining features for our users**_. Following the same spirit we decided that it was important to upgrade to the latest MUI v5.
_**Continuing with this spirit we decided**_ that it was important to upgrade to the lastest MUI V5